e2e/tests/components/sliders/sliders.e2e-spec.ts

```diff
@@ -344,6 +344,20 @@ describe('Sliders', () => {
 
     });
 
+    it('should allow dragging of the lower thumb when both values are set to the max', async () => {

```

Comment:
We're going to need more test coverage for this. I linked a sandbox previously with some test cases, that should be a minimum. You can set up a new test page with more numerical inputs to configure min and max and so on.